tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -69,23 +75,33 @@ function install_packages() {
     radvd \
     sharutils genisoimage \
     strongswan libcharon-extra-plugins libstrongswan-extra-plugins strongswan-charon strongswan-starter \
-    virt-what open-vm-tools qemu-guest-agent hyperv-daemons
+    virt-what open-vm-tools qemu-guest-agent hyperv-daemons cloud-guest-utils \
+    conntrack apt-transport-https ca-certificates curl gnupg  gnupg-agent software-properties-common
+
+  apt-get install -y python3-json-pointer python3-jsonschema cloud-init
 
   # python2-netaddr workaround
   wget https://github.com/shapeblue/cloudstack-nonoss/raw/main/python-netaddr_0.7.19-1_all.deb
   dpkg -i python-netaddr_0.7.19-1_all.deb
 
-  apt-get -y autoremove --purge
-  apt-get clean
-  apt-get autoclean
+  apt_clean
 
-  #32 bit architecture support for vhd-util: not required for 32 bit template
+  # 32 bit architecture support for vhd-util
   if [ "${arch}" != "i386" ]; then
     dpkg --add-architecture i386
     apt-get update
     ${apt_get} install libuuid1:i386 libc6:i386
   fi
 
+  # Install docker and containerd for CKS
+  curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+  apt-key fingerprint 0EBFCD88
+  add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+  apt-get update
+  ${apt_get} install docker-ce docker-ce-cli containerd.io
```

Comment:
maybe we don't need CLI? Or both docker-ce, we use ctrctl to import images? If not now - in future phase?

Reply:
+1 - will work on it in the next phase